app/services/case_filter/case_status_filter.rb

```diff
@@ -0,0 +1,59 @@
+module CaseFilter
+  class CaseStatusFilter < CaseMultiChoicesFilterBase
+
+    def self.filter_attributes
+      [:filter_status]
+    end
+
+    def get_available_choices
+      {
+        :filter_status => 
+          {'open'   => I18n.t('filters.statuses.open'),
+          'closed' => I18n.t('filters.statuses.closed')}
+      }
+    end
+
+    def call
+      filter_status(@records)
+    end
+
+    def crumbs
```

Comment:
I think we may be able to extract `def crumbs` to a class lower down the hierarchy. Seems to be of a similar shape in all the subclasses and relies on a method being called on "`@query`" - I think we could add some polymorphism here by passing in the differing method as a symbol. Or decorating the query object and returning the correct data via the same interface at a per class level.